src/main/kotlin/com/openlattice/hazelcast/serializers/NoOpSelfRegisteringStreamSerializer.kt

```diff
@@ -0,0 +1,16 @@
+package com.openlattice.hazelcast.serializers
+
+import com.hazelcast.nio.ObjectDataOutput
+import com.kryptnostic.rhizome.pods.hazelcast.SelfRegisteringStreamSerializer
+import com.openlattice.hazelcast.StreamSerializerTypeIds
+
+abstract class NoOpSelfRegisteringStreamSerializer<T>: SelfRegisteringStreamSerializer<T> {
+    override fun getTypeId(): Int {
+        return StreamSerializerTypeIds.NO_SERIALIZATION_ENTRY_PROCESSOR.ordinal
```

Comment:
looks like this is making hazelcast unhappy (since multiple stream serializers are getting registered for the same type id)

Reply:
was this fixed?